api-reference-v2/api-reference/organization/organization--create.mdx

```diff
@@ -0,0 +1,3 @@
+---
+openapi: post /organization
```

Comment:
V2 will have v2 prefix

Reply:
In other places too

Reply:
/organization can be considered as a new endpoint introduced in V2.